### DISCUSSION TITLE
notes?

### DISCUSSION THREAD

**Discussion by oldfart101** [Lissy93/dashy#414] [General]
any chance of a 'notes' window attached to an item. 
For my use case, the description text isn't enough.
The notes could be protected by user permissions?
thanks for your help so far

**Comment by lissy93** [comment]
You can check out the current [Road Map](https://github.com/Lissy93/dashy/discussions/405) to see what's planned up next.
Haven't got any immediate plans to add a notes attachment quite yet, but if you wanted to submit a PR, I'd be happy to review that :)
